platform/iphone/godot_view.mm

```diff
@@ -361,10 +361,11 @@ - (void)godotTouchesMoved:(NSSet *)touches withEvent:(UIEvent *)event {
 			CGPoint touchPoint = [touch locationInView:self];
 			CGPoint prev_point = [touch previousLocationInView:self];
 			CGFloat force = touch.force;
-			// Vector2 tilt = touch.azimuthUnitVector;
+			CGVector tilt = [touch azimuthUnitVectorInView:self];
```

Comment:
`azimuthUnitVectorInView` is not exactly the same as the tilt values on the other platforms (which can be in the `-1 ... 1` range on both axis, like - https://developer.apple.com/documentation/appkit/nsevent/1534226-tilt?language=objc).

Reply:
Oh nice, thanks. I will check how to use this.